### DISCUSSION TITLE
How do I set form to empty object?

### DISCUSSION THREAD

**Discussion by madsh93** [inertiajs/inertia#1287] [Help]
I'm populating a form without knowing the properties, thus I create an empty object with useForm

```js
const formContainer = useForm<{
  [key: string]: any;
}>({});
```

After populating the formContainer, how can I reset the formContainer to an empty object? `formContainer.reset()` does not seem to work without predefined properties.

**Comment by juanvillans** [comment]
Could you let me know if you found a solution? I want to set it to empty all the values of the object's properties
